### DISCUSSION TITLE
Expand the area of quick scroll in lazer

### DISCUSSION THREAD

**Discussion by Nekomosh004** [ppy/osu#19697] [Ideas] [answered]
There's this amazing option we use to quickly scroll in the Song Select menu, without using the scroll bar or mouse scroll. We hover the mouse cursor on an empty area of the Song Select menu, then right-click to quickly scroll (jump).

Here's a video of how this works in Stable/CE vs lazer:

https://user-images.githubusercontent.com/82471957/184023162-5a569ff7-0242-4de6-a384-ba15e2e64df7.mp4

Compared to lazer, this option can be done in stable/CE in quite big area of the Song Select menu. Screenshot below shows a divider (red line) of how big (approximately) the area this option can be done in lazer (top) and Stable/CE (bottom). 

![osu](https://user-images.githubusercontent.com/82471957/184021243-b72d475d-e7f9-47cb-a52d-923da8dc4a17.jpg)

I hope the quick scroll area for lazer can be expanded, at most before the ranking panels on the left.
Thank you for considering! :D

**Comment by TheFantasticLoki** [comment, marked as answer]
This is a duplicate of #17500 and #2238 but yes this still needs to be addressed as I cannot believe it hasn't been already.

**Reply by Nekomosh004** [reply]
Unaware this was a duplicate, thanks! ❤️
